schema.sql

```diff
@@ -1,8 +1,13 @@
 DROP TABLE IF EXISTS Users CASCADE;
 DROP TABLE IF EXISTS UserBooks CASCADE;
 DROP TABLE IF EXISTS Books CASCADE;
-DROP TABLE IF EXISTS Articles CASCADE;
-DROP TABLE IF EXISTS UserArticles CASCADE;
+
```

Comment:
I wouldn't remove the User tables since our vision for the app was to have user specific references, but I would rename the books table to references.